### ISSUE TITLE
Implement a Ghost model in pacman PO

### ISSUE THREAD

**Issue by Squillyprice01** [schrum2/MM-NEAT#175]

**Comment by schrum2** [comment]
Include more details. We discussed several important issues with this on
the board today, and I would like a record of them in GitHub

**Comment by Squillyprice01** [comment]
As the last two commits state, we know have a working ghost model. It is parameterized by the following parameters:

- probabilityThreshold is a double parameter that stands as the minimum probability of a ghost location that we keep track of and that we draw. 

- useGhostModel is a boolean parameter that decides whether or not we even instantiate all of the machinery for a ghost model.

- drawGhostPredictions is another boolean parameter that decides whether or not we draw the probable ghost locations onto the game board.

Integrating the ghostModel into sensors still needs to be done. Yesterday we discussed how to treat these ghost probabilities. We decided that the closest potential ghost is the one we pay attention to and ignore the others. I would like to add that the higher we set the probabilityThreshold, the less red ghosts we have moving in every which way.
